modules/ROOT/pages/analytics-event-api.adoc

```diff
@@ -102,13 +119,22 @@ Example output for this command is:
 
 === Supporting Environments
 
-For organizations that have enabled environments, API events queried using the approach described above will belong exclusively to the Unclassified" environment. To request events for APIs in a specific environment, select the environment in the environment selector and create the report that you want to run. Reports are stored per-environment, so different reports will exist in different environments. Copy the report URL from the UI, and note that in addition to the Organization ID, the URL also includes `/environment/{Environment ID}`. Anypoint Platform assigns the unique and immutable Organization ID and Environment IDs at organization and environment creation times.
+For organizations that have enabled environments, API events queried using the approach described in the previous section belongs exclusively to the "Unclassified" environment. 
+
+To request events for APIs in a specific environment:
+
+. Select the environment in the Environment selector and create the report that you want to run. 
++
+Reports are stored based on the environment. Therefore, different reports exist in different environments. 
+. Copy the report URL from the UI.
++
+Note that in addition to the organization ID, the URL also includes `/environment/{Environment ID}`. Anypoint Platform assigns a unique ID to an organization and an environment when they are created. For example, an API ID in QA environment will be different from its ID in Production.
 
-The addition of the Environment ID to the report URL is all you need to access events in the specified environment. API IDs are different in different environments: An API's ID in QA will be different from its ID in Production, for example.
+To access events for a specific environment, append the environment ID to the report URL. 
```

Comment:
Is this supposed to be the final step in the task?
This task is mostly conceptual information, which is not optimal. 
Review and shorten to the step alone (as much as reasonable) each step in the task of requesting events for APIs in a specific environment.

Reply:
No change required.